package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gocrisp/store-locator",
-  "version": "0.5.6",
+  "version": "0.5.8",
```

Comment:
This wasn't really necessary but the github actions publish will have an issue if I don't bump this since it has issues committing the change back to the repo right now and I haven't figured out the right security settings.